src/createServer.js

```diff
@@ -1,3 +1,35 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase/convertToCase');
+const validation = require('./validation');
+
+module.exports.createServer = () => {
+  const server = http.createServer((req, res) => {
+    const normalizedURL = new URL(req.url, `http://${req.headers.host}`);
+    const text = normalizedURL.pathname.replace('/', '');
+    const caseName = normalizedURL.searchParams.get('toCase');
+
+    res.setHeader('Content-type', 'application/json');
+
+    const errors = validation(text, caseName);
+
+    if (errors.length) {
+      res.status = 400;
```

Comment:
The correct property to set the status code is `statusCode`, not `status`. Correct this to `res.statusCode = 400;`